ui/definitions/mutualfunds.d.ts

```
@@ -1,55 +1,55 @@
-  sub: string;
+  main: string
+  sub: string
 }
 export interface Folio {
-  folio: String;
-  invested: number;
-  units: number;
-  value: number;
-  // eslint-disable-next-line camelcase
-  avg_nav: number;
+  folio: String
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)